### PR TITLE
Added LICENSE file and associated file headers

### PR DESCRIPTION
**Pull request by Davo36**
Added GNU v3.0 license file to the repository.

Also added a file header to the 2 python files.

### CODE REVIEW

File: 3g-watchdog [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
```
Comment (mjs) [inline 3g-watchdog]: The header looks great but this isn't a valid comment for a Python program! You'll need to change it to a `#` or triple quote style comment (https://twitter.com/gvanrossum/status/112670605505077248)

File: 3g-watchdog [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+    
 #!/usr/bin/python3 -u
```
Comment (mjs) [inline 3g-watchdog]: This needs to be the very first line or it won't work. The license header will have to come straight after.

File: 3g-watchdog_test.py [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
```
Comment (mjs) [inline 3g-watchdog_test.py]: Same issue with the comments

**Review by mjs — COMMENTED** [review]
Thanks! Just a few things to sort out.

Pro tip: if you add something like `Closes #7` or `Fixes #7` to the PR description, that issue will be automatically closed when the PR is merged.

**Comment by Davo36** [comment]
Hi Menno, sorry about that, was in Go mode.

Which raises the issue of testing I guess.  I will need to know what to do there too.

**Review by mjs — APPROVED** [review]
Nice!

**Comment by mjs** [comment]
There are some tests but we don't have them running automatically for pull requests for this repository. We do have checks for some of our other repositories though (and you can't merge a PR until the tests pass).

**Comment by Davo36** [comment]
Oh Ok, that sounds like a good idea.

I'll install PyTest and have a play with it.


David


____________________________________________________________________________________
David Blake
Timepiece Investments Limited
36 Wood Bay Road,
Titirangi,
Auckland,
New Zealand
Ph:   +64-9-816-9570
Mobile:  +64-21-725-909

**Comment by mjs** [comment]
@Davo36 good idea. pytest is a really nice test framework. I think you'll like it.

We're using Python 3 for everything so make sure you're running code and tests using 3.5 or later.

**Pull request merged**